src/F2_autoreport.py

```diff
@@ -510,9 +510,8 @@ def create_app(results_json):
     warnings_dict = {}
     errors_dict = {}
 
-    log_file = os.path.join(OUTPUT_FOLDER, "mvs_logfile.log")
-    # log_file = "/home/mr/Projects/mvs_eland/MVS_outputs/mvs_logfile.log"
-    print(log_file)
+    # log_file = os.path.join(OUTPUT_FOLDER, "mvs_logfile.log")
```

Comment:
If this is the commit you didn't want, you can type `git reset --hard 16794b890570b94ed15b43e32aeabbbd060f931d` (which is the commit number of the previous commit), then you need to `git push -f`

Reply:
Just did it with GIT Kraken

I first undid the commit and then force pushed it.